pdns/lua-recursor4.cc

```diff
@@ -343,6 +343,12 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerFunction("getRecords", &DNSQuestion::getRecords);
   d_lw->registerFunction("setRecords", &DNSQuestion::setRecords);
 
+  d_lw->writeFunction("addPolicyTag", [](vector<string>* tags, const string& tag) {
+      if (tags) {
+        tags->push_back(tag);
+      }
+    });
+
```

Comment:
This weird idiom (If I see it correctly, inside `gettag` I am supposed to call `addPolicyTag(policytags, 'foo')`) is because we don't have any state object to speak of, like dq?